### ISSUE TITLE
"EndlessPiece" has missing script

### ISSUE THREAD

**Issue by rhys-vdw** [dentedpixel/LeanTween#163]
Leantween v2.50

> You are trying to replace or create a Prefab from the instance 'EndlessPiece' that references a missing script. This is not allowed.
Please change the script or remove it from the GameObject.

![image](https://user-images.githubusercontent.com/822105/115322343-35eaa280-a1c9-11eb-85f2-0a598a480d1f.png)

As a side note, please consider putting examples folder in a `.unitypackage` so that each developer using your project doesn't need to cop the asset import time.

**Comment by OblicaStudio** [comment]
I was having the same problem. When I imported LeanTween it showed above error on compilation. I was not able to use this package as it was not properly imported. To fix it, open any LeanTween script in the Visual Studio (Double click the script in the Unity Editor). Once VS project opens build the solution. Come back to Unity Editor and the error has gone.

**Comment by basisbit** [comment]
there has been no development on LeanTween since 2018. The project can be assumed dead. Check the forks to find something more recent with bugfixes or similar.

**Comment by OblicaStudio** [comment]
Yeah, and now its support site (DentedPixel) is also dead. Any idea where I can access LeanTween documention?

**Comment by basisbit** [comment]
have you checked waybackmachine / web archive?

**Comment by rhys-vdw** [comment]
> there has been no development on LeanTween since 2018. The project can be assumed dead. Check the forks to find something more recent with bugfixes or similar.

Yeah, I switched to DOTween, much better.

**Comment by OblicaStudio** [comment]
> have you checked waybackmachine / web archive?

Yeah, I have checked it but nothing there. I have now emailed to developer hoping his email is still active.
